specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-11-01/webapplicationfirewall.json

```diff
@@ -270,8 +270,15 @@
         "etag": {
           "type": "string",
           "description": "Gets a unique read-only string that changes whenever the resource is updated."
+        },
+        "sku": {
+          "description": "The pricing tier of web application firewall policy.",
+          "$ref": "#/definitions/Sku"
         }
       },
+      "required": [
+        "sku"
+      ],
```

Comment:
Add a required property is a breaking change, even in a new version. according to breaking change policy: http://aka.ms/bcforapi
can it be optional?

Reply:
Thank you for pointing that out. I have changed Sku to optional.